# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/performance-timeline/po-observe-type.any.js.ini
# Commit: e04ef5171459
# Full Hash: e04ef517145991e2aee797e935f43b7be35ff7aa
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-09 09:54:40
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D25426
# ==============================================================================

diff -r ffaff6fb1fe3 -r e04ef5171459 testing/web-platform/meta/performance-timeline/po-observe-type.any.js.ini
--- a/testing/web-platform/meta/performance-timeline/po-observe-type.any.js.ini	Mon Apr 08 19:00:31 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,33 +0,0 @@
-[po-observe-type.any.html]
-  [Calling observe() with entryTypes and then type should throw an InvalidModificationError]
-    expected: FAIL
-
-  [Calling observe() with type and then entryTypes should throw an InvalidModificationError]
-    expected: FAIL
-
-  [Calling observe() with type and entryTypes should throw a SyntaxError]
-    expected: FAIL
-
-  [Passing in unknown values to type does throw an exception.]
-    expected: FAIL
-
-  [observe() with different type values stacks.]
-    expected: FAIL
-
-
-[po-observe-type.any.worker.html]
-  [Calling observe() with entryTypes and then type should throw an InvalidModificationError]
-    expected: FAIL
-
-  [Calling observe() with type and then entryTypes should throw an InvalidModificationError]
-    expected: FAIL
-
-  [Calling observe() with type and entryTypes should throw a SyntaxError]
-    expected: FAIL
-
-  [Passing in unknown values to type does throw an exception.]
-    expected: FAIL
-
-  [observe() with different type values stacks.]
-    expected: FAIL
-